sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/DeclareTypesInNamespacesTest.cs

```diff
@@ -54,11 +54,17 @@ public void DeclareTypesInNamespaces_CS_Before8()
 
         [TestMethod]
         [TestCategory("Rule")]
-        public void DeclareTypesInNamespaces_VB()
-        {
+        public void DeclareTypesInNamespaces_CS_AfterCSharp9() =>
+            Verifier.VerifyAnalyzer(@"TestCases\DeclareTypesInNamespaces.AfterCSharp9.cs",
```

Comment:
Why not follow the same convention with `CSharp9`?

Reply:
to have the same pattern with the other files
`DeclareTypesInNamespaces_CS_Before8`
`DeclareTypesInNamespaces_CS_After8`

same for the test file name